_projects/2019/sunpy/mag_field_draft.md

```diff
@@ -0,0 +1,67 @@
+---
+name: Expand the scope of solarbextrapolation.
+desc: Add more numerical models and generalize magnetic field extrapolation to spherical coordinates.
+requirements:
+ - Knowledge of vector calculus and differential equations
+ - Familiar with numerical methods
+ - Some familiarity with electromagnetism, MHD, potential/force-free magnetic fields (optional)
+difficulty: intermediate
+issues:
+ - https://groups.google.com/forum/#!topic/sunpy/ZLu_HoX_lbc
+ - https://github.com/sunpy/solarbextrapolation/issues/12
+mentors:
+ - FIND SOMEONE MAYBE
```

Comment:
Given @dstansby and @wtbarnes are in the Thesis pit, I think we might have to pass on this one this year, unless someone steps up to mentor it. @Alex-Ian-Hamilton ?

Reply:
Probably best to drop it for this year.

Reply:
I'd be willing to mentor this.

Reply:
Put me down as a second :roll_eyes: